build_system/seqan3-config.cmake

```diff
@@ -136,45 +136,36 @@ macro (seqan3_config_error text)
 endmacro ()
 
 # ----------------------------------------------------------------------------
-# Detect if we are a clone of repository and if yes auto-add submodules
+# Find SeqAn3 include path
 # ----------------------------------------------------------------------------
+
 # Note that seqan3-config.cmake can be standalone and thus SEQAN3_CLONE_DIR might be empty.
 find_path (SEQAN3_CLONE_DIR NAMES build_system/seqan3-config.cmake HINTS "${CMAKE_CURRENT_LIST_DIR}/..")
+find_path (SEQAN3_INCLUDE_DIR NAMES "seqan3/version.hpp" HINTS "${SEQAN3_CLONE_DIR}/include")
```

Comment:
Does `${SEQAN3_CLONE_DIR}/include` lead to an error if CLONE_DIR is empty?

Reply:
no, it is only a hint. if he doesn't find it; he will not find it :)